imports/plugins/included/taxes-avalara/client/containers/avalaraSettingsFormContainer.js

```diff
@@ -0,0 +1,153 @@
+import _ from "lodash";
+import { compose, withProps } from "recompose";
+import { composeWithTracker, registerComponent } from "@reactioncommerce/reaction-components";
+import { Meteor } from "meteor/meteor";
+import { Reaction, i18next } from "/client/api";
+import { AvalaraPackageConfig } from "../../lib/collections/schemas";
+import { AvalaraSettingsForm } from "../components";
+import { Logs as LogSchema } from "/lib/collections/schemas/logs";
+import { Countries } from "/client/collections";
+
+/**
+ * @file This is a container for AvalaraSettingsForm.
+ * @module avalaraSettingsFormContainer
+ */
+
+const handlers = {
+  /**
+    * handleSubmit
+    * @method
+    * @summary event handler for when new Avalara settings are submitted.
+    * @param {Object} event - event info.
+    * @param {Object} changedInfo - info about the new Avalara settings.
+    * @param {String} targetField - where to save the new settings in the Avalara Package.
+    * @return {null} - returns nothing
+    */
+  handleSubmit(event, changedInfo, targetField) { // eslint-disable-line no-unused-vars
+    if (!changedInfo.isValid) {
+      return;
+    }
+    Meteor.call("package/update", "taxes-avalara", "settings", changedInfo.doc.settings, (error) => {
+      if (error) {
+        Alerts.toast(
+          i18next.t("admin.update.avalaraUpdateFailed", { defaultValue: "Failed to update Avalara settings." }),
+          "error"
+        );
+        return;
+      }
+      Alerts.toast(
+        i18next.t("admin.update.avalaraUpdateSucceeded", { defaultValue: "Avalara settings updated." }),
+        "success"
+      );
+    });
+  },
+
+  handleTestCredentials(event, settings) {
+    Meteor.call("avalara/testCredentials", settings.avalara, function (error, result) {
+      if (error && error.message) {
+        return Alerts.toast(`${i18next.t("settings.testCredentialsFailed")} ${error.message}`, "error");
+      }
+      try {
+        const statusCode = _.get(result, "statusCode");
+        const connectionValid = _.inRange(statusCode, 400);
+        if (connectionValid) {
+          return Alerts.toast(i18next.t("settings.testCredentialsSuccess"), "success");
+        }
+        return Alerts.toast(i18next.t("settings.testCredentialsFailed"), "error");
+      } catch (e) {
```

Comment:
Don't use single letter variables. Always use `error`. If you can't because of scope then use `err`